frontend/src/stores/useSelectionStore.ts

```diff
@@ -1,8 +1,21 @@
-import create from 'zustand'
-
+import create, { SetState } from 'zustand'
 import { useProjectStore } from '/src/stores'
+import { AutomataState, AutomataTransition, ProjectComment } from 'src/types/ProjectTypes'
+
+type SelectionStore = {
+  selectedStates: AutomataState[],
+  selectedTransitions: AutomataTransition[],
+  selectedComments: ProjectComment[],
```

Comment:
```suggestion
  selectedStates: number[],
  selectedTransitions: number[],
  selectedComments: number[],
```
These get mapped to ID's later which are numbers

Reply:
This change will cause errors in the other stuff, make sure to fix those also :+1:

Reply:
Good pickup will shuffle it around